ihmc-avatar-interfaces/src/main/java/us/ihmc/avatar/networkProcessor/kinematicsStreamingToolboxModule/KSTStreamingState.java

```diff
@@ -117,7 +116,7 @@ public class KSTStreamingState implements State
 
    private final YoDouble inputWeightDecayFactor;
    private final YoInteger numberOfDecayingInputs = new YoInteger("numberOfDecayingInputs", registry);
-   private final YoBoolean isDemonstrationEpisode = new YoBoolean("isDemonstrationEpisode", registry);
+   private final YoInteger demonstrationTaskID = new YoInteger("demonstrationTaskID", registry);
```

Comment:
Change boolean to int so the operator can annotate more than one task at a time. Really useful when doing the inverse task.